bgpd/bgp_bmp.c

```diff
@@ -438,6 +438,7 @@ static struct stream *bmp_peerstate(struct peer *peer, bool down)
 	monotime_to_realtime(&uptime, &uptime_real);
 
 #define BGP_BMP_MAX_PACKET_SIZE	1024
+#define BMP_PEERUP_INFO_STRING	0
```

Comment:
The naming IMO isn't right (maybe something like BMP_INFO_TYPE_PEERUP?). It defines a type, correct?

Reply:
I meant that `TYPE` as others would be more readable.

Reply:
oops i meant to put BMP_PEERUP_INFO_TYPE_STRING similar to BMP_INIT_INFO_TYPE_XXXX
you could have multiple different info types for bmp peerup messages so what about BMP_PEERUP_INFO_TYPE_STRING ?

Reply:
:+1:

Reply:
done 🙂